src/components/events/create-event/FormContainer.js

```diff
@@ -134,8 +135,14 @@ const FormContainer = () => {
               })
               .catch((err) => console.log(err));
           } else {
+            const timeObject = parseTime(
+              new Date(`${values.date} ${values.startTime}`).getTime(),
+              values.startTime,
+              values.endTime
+            );
             const newEvent = {
               ...values,
+              date: timeObject.formattedDate,
```

Comment:
@kyle-richardson is this `date:` supposed to be `startTime:`?

Reply:
good catch.  This and other changes to the actual create event part of the date changes will be handled by @patrick-replogle in a separate pull request.